lib/morandi/version.rb

```diff
@@ -1,3 +1,3 @@
 module Morandi
-  VERSION = '0.10.3'.freeze
+  VERSION = '0.10.4'.freeze
```

Comment:
The version actually adds a new feature i.e. setting quality using options, so `0.11.0` would be a more appropriate version number